api_grpc/worker.go

```diff
@@ -315,7 +317,7 @@ func (s server) NotifyStreamStarted(ctx context.Context, request *pb.StreamStart
 		default:
 			dao.SaveCOMBURL(&stream, request.HlsUrl)
 		}
-		api.NotifyViewersLiveState(stream.Model.ID, true)
+		//api.NotifyViewersLiveState(stream.Model.ID, true)
```

Comment:
Bummer, I guess we could move this grpc stuff into the API package to resolve this.

Reply:
I had the same idea, but I was not sure about it. I guess a package for one file is not optimal anyways.